test/torch/model/test_mqf2_modules.py

```diff
@@ -44,16 +44,19 @@ def test_mqf2_modules(
 
     distr_output = MQF2DistributionOutput(prediction_length)
 
-    model = MQF2MultiHorizonModel(
-        freq="1H",
-        context_length=context_length,
-        prediction_length=prediction_length,
-        num_feat_dynamic_real=num_feat_dynamic_real,
-        num_feat_static_real=num_feat_static_real,
-        num_feat_static_cat=num_feat_static_cat,
-        cardinality=cardinality,
-        distr_output=distr_output,
+    lightning_module = MQF2MultiHorizonLightningModule(
+        {
+            "freq": "1H",
+            "context_length": context_length,
+            "prediction_length": prediction_length,
+            "num_feat_dynamic_real": num_feat_dynamic_real,
+            "num_feat_static_real": num_feat_static_real,
+            "num_feat_static_cat": num_feat_static_cat,
+            "cardinality": cardinality,
+            "distr_output": distr_output,
+        }
```

Comment:
you don't like the `dict(...)` notation, do you?

Reply:
I think `{...}` is visually easier to understand.